pkg/azureconstants/azure_constants.go

```diff
@@ -134,6 +134,8 @@ const (
 	RequestTimeFormat              = time.RFC3339Nano
 	RequesterKey                   = "disk.csi.azure.com/requester-name"
 	WorkflowKey                    = "disk.csi.azure.com/requester-name"
+	VolumeAttachRetryAnnotation    = "disk.csi.azure.com/volume-attach-retry"
+	VolumeAttachRetryCount         = "disk.csi.azure.com/volume-attach-retry-count"
```

Comment:
Instead of keeping two annotations for retry, we could just keep the `volume-attach-retry-count`. When retry is needed, we could increment the count.

Reply:
Actually, since we have two controllers for azvolumeattachments, having two different annotations might make it easier to coordinate the operations.